frontend/package.json

```diff
@@ -84,11 +83,11 @@
     "patch-id": "bash -c \"cp patch/imagery.min.json public/static/id/data\"",
     "patch-rapid": "bash -c \"cp patch/rapid-imagery.min.json public/static/rapid/data/imagery.min.json\"",
     "preparation": "bash -c \"if (test -a ../tasking-manager.env); then grep -hs ^ ../tasking-manager.env .env.expand > .env; else cp .env.expand .env; fi\"",
-    "start": "npm run preparation && npm run copy-static && npm run patch-id && npm run patch-rapid && react-scripts start",
-    "build": "npm run preparation && npm run update-static && npm run patch-id && npm run patch-rapid && react-scripts build",
+    "start": "npm run preparation && npm run copy-static && npm run patch-id && npm run patch-rapid && craco start",
+    "build": "npm run preparation && npm run update-static && npm run patch-id && npm run patch-rapid && craco build --stats",
```

Comment:
To achieve quicker and leaner production bundles, wouldn't it be desirable to omit the `--stats` flag?

Reply:
> To achieve quicker [...] production bundles, wouldn't it be desirable to omit the --stats flag?

Surprisingly, no.
Here are some test build times (using `time`, and specifically the total time elapsed). Starting with run 22, the runs were with all other applications closed _and_ the computer locked.

<details><summary>Runtime statistics for <code>--stats</code> and no <code>--stats</code></summary>

Run | No stats (s) | Stats (s)
-- | -- | --
1 | 129.38 | 187.45
2 | 110.25 | 76.32
3 | 107.88 | 76.46
4 | 96.8 | 81.37
5 | 97.73 | 79.5
6 | 131.19 | 79.67
7 | 111.21 | 79.06
8 | 95.57 | 80.03
9 | 92.99 | 78.9
10 | 89.16 | 80.49
11 | 93.77 | 81.54
12 | 96.78 | 80.46
13 | 112.91 | 80.05
14 | 95.33 | 80.15
15 | 86.28 | 81.81
16 | 91.38 | 81.74
17 | 88.22 | 81.17
18 | 96.19 | 80.85
19 | 96.58 | 78.06
20 | 93.15 | 80.99
21 | 91.93 | 79.3
22 | 84.5 | 75.25
23 | 87.12 | 79.56
24 | 87.97 | 78.61
25 | 84.9 | 79.1
26 | 84.71 | 79.02
27 | 85.36 | 77.64
28 | 82.91 | 57.84

</details>

> To achieve [...] leaner production bundles, wouldn't it be desirable to omit the --stats flag?

`build` is 339M for `--stats` with 107M from the stats file, and 227M without the `--stats`. With that said, the numbers really need to be compared over multiple runs. And isn't what would reach the users _anyway_. For reference, it is 228M with current `develop`.

Reply:
But isn't it evident from the table that excluding the "--stats" flag generally led to shorter build times?

Could you kindly clarify why you mentioned that excluding the "--stats" flag might not be desirable? I value your insights and would appreciate further clarification.

Reply:
> But isn't it evident from the table that excluding the "--stats" flag generally led to shorter build times?

That wasn't the case. I probably should have flipped the description (`--stats` and no `--stats` -> no `--stats` and `--stats`) to match the table headers.

The average for no `--stats` was 96.5s and for `--stats` it was 82.5s. The median for no `--stats` was 93.46s and the median for `--stats` was 79.6s.

It _does_ produce a slightly larger `build` directory, most of which is from the `bundle-stats.json` file (which doesn't need to be served). I should probably do another 10 runs of `--stats` and no `--stats` to see how the `build` directory size changes and whether or not it is _consistent_ between runs.

Reply:
I was primarily only trying to understand whether including the `--stats` flag would make a significant difference. Considering the insights you've provided and the analysis conducted, it seems reasonable to proceed with merging the pull request as it is. Does anything need to be addressed to prevent the stats file from being served?

Thank you for your valuable contributions to this conversation. Are there any additional considerations before merging?

Reply:
> Does anything need to be addressed to prevent the stats file from being served?

Probably. I'll take a quick look into it. Realistically, we could just do an `rm` in the CI prior to deployment. But that probably isn't the best option. Something that might be better is a check, e.g. `if [ "${NODE_ENV}" == "PRODUCTION" ] ; then craco build; else craco build --stats; fi`. That works on mac/linux, but might not work on Windows.

With that said, I _think_ there is a ticket open for switching to Terraform and there is a repo for that.